tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml

```diff
@@ -18,7 +18,7 @@ spec:
           katacontainers.io/kata-runtime: cleanup
       containers:
       - name: kube-kata-cleanup
-        image: quay.io/kata-containers/kata-deploy:latest
+        image: quay.io/kata-containers/kata-deploy:2.3.0-rc0
```

Comment:
I would like to know what's the motivation for this change.

Reply:
@amshinde, this is similar to the reply above, more info in https://github.com/kata-containers/kata-containers/pull/2453 (and its associated issue)
Please, let me know if there's something that's not clear there.

Reply:
See 43a72d76e2520ae044335b9f1dbf54c12363f54b